### ISSUE TITLE
Remove GUI Dependency From OoTRandomizer.py

### ISSUE THREAD

**Issue by timmonfette1** [AmazingAmpharos/OoT-Randomizer#40]
This is an issue (albeit a slight one), but it's also a feature request.

I think the `OoTRandomizer.py` script should be re-worked to not include any dependency being able to load/run the GUI.  I think `OoTRandomizer.py` should be a pure command line interface and the `GUI.py` script should be a pure Graphical User Interface.  Having `OoTRandomizer.py` support the `--gui` flag caused me issues with trying to run it (plus I'm not a fan of supporting 2 different types of interfaces in one script/program).

I'm not sure if `OoTRandomizer.py` was written this way because of how you're distributing the bundled code, but having it depend on being able to load the GUI seems silly since it's mostly a CL interface to the randomizer.  That's why I think it needs a little tweaking to better separate the concerns of what interface a user wants to use.

My Linux distro doesn't support tkinter easily so there's no way I can run the GUI right now (I don't want to anyway).  However, since `OoTRandomizer.py` depends on being able to import guiMain from `GUI.py`, it causes the script to fail when I try to run it because it can't import tkinter.

This is hardly a huge issue since all I had to do was just modify about 4 lines of code to remove the import statement (line 9)
```
#from Gui import guiMain
```
Remove every reference to guiMain (lines 73 and 86)
```
#guiMain()
#guiMain(args)
```
Add a None to an if block (line 86). This chunk could be removed (and should be if the `--gui` flag is dropped) and the first `elif` becomes the leading `if`, but for simplicity I just did this.
```
if args.gui:
        #guiMain(args)
        None
```
Having to do this was a minor annoyance since I didn't expect the main script depend on the GUI especially because the GUI can run as a standalone if someone wants to use it.

Obviously this might have an impact on how you're bundling/distributing the code.  The first, and maybe easiest, solution I thought of was to distribute 2 installers/distributions - one for the command line interface (`OoTRandomizer.py` with no GUI flag support) and a second one for the graphical interface (`GUI.py`).  This way the user can decide which one they want to download and use, or if someone does a clone of the repo, they can use either script to run the interface of their choice.

**Comment by jamesmcm** [comment]
Yeah, it seems to be because the bundled build is running OoTRandomizer.py directly as the exe.

A quick hack would be to make the GUI import also conditional on is_bundled()

Also for future reference, I think it's best to use pass where you used None in the last block.

**Comment by AmazingAmpharos** [comment]
I'll probably be looking into this when I'm bundling up 2.0 for official release.
